addons/auth_signup/auth_signup_data.xml

```diff
@@ -50,6 +50,11 @@
                     <p>
                         You have been invited to connect to "${object.company_id.name}" in order to get access to your documents in Odoo.
                     </p>
+                    % if ctx.get("custom_message"):
+                    <p>
+                        ${ctx.custom_message}
```

Comment:
We are not saving message in database. so i sent it through context.
![download 8](https://cloud.githubusercontent.com/assets/7629089/9112092/174dca84-3c6a-11e5-9eee-126bebaeb134.png)